### ISSUE TITLE
radius graph defaults

### ISSUE THREAD

**Issue by nec4** [torchmd/torchmd-net#21]
Hello everyone! The tools are great so far. I noticed that there is a small but important corner case for the usage of `radius_graph` from `torch_cluster`. When searching for neighbors, the behavior is governed by the cutoff distance `r` and `max_num_neighbors` (see docs here: https://github.com/rusty1s/pytorch_cluster#radius-graph). The latter is set to a maximum of 32 neighbors for each node. If, for example, the user inputs a large cutoff distance intending to return all neighbors, they will still be truncated at a maximum of 32 even if the user expects more. Furthermore, I'm not sure how `radius_graph` decides to reject extra neighbors, or how example shuffling during training affects this  - for my usage case it seems to make a big difference in the training and inference. Because the SchNet philosophy is to operate on the notion of cutoff distances, not maximum neighbors, would it make sense to add a kwarg to the `TorchMD_GN.__init__()` raise the limit of the max neighbors for this operation? 

Of course, most users probably will not run into this problem if they stick to small cutoffs because they will never hit the upper neighbor ceiling. However, I would be happy to branch, implement this, write tests, and make a PR if it seems like a good idea.
